components/prize_pool/wikis/ageofempires/prize_pool_custom.lua

```diff
@@ -0,0 +1,81 @@
+---
+-- @Liquipedia
+-- wiki=ageofempires
+-- page=Module:PrizePool/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+local Opponent = require('Module:Opponent')
+
+local PrizePool = Lua.import('Module:PrizePool', {requireDevIfEnabled = true})
+
+local LpdbInjector = Lua.import('Module:Lpdb/Injector', {requireDevIfEnabled = true})
+local CustomLpdbInjector = Class.new(LpdbInjector)
+
+local CustomPrizePool = {}
+
+local TIER_VALUE = {10, 6, 4, 2}
+
+-- Template entry point
+function CustomPrizePool.run(frame)
+	local args = Arguments.getArgs(frame)
+	args.opponentLibrary = 'Opponent/Custom'
+	args.loadFlags = true
+	args.loadTeams = true
+
+	local prizePool = PrizePool(args)
+		:create()
+
+	prizePool:setLpdbInjector(CustomLpdbInjector())
+
+	return prizePool:build()
+end
+
+function CustomLpdbInjector:adjust(lpdbData, placement, opponent)
+	lpdbData.weight = CustomPrizePool.calculateWeight(
+		lpdbData.prizemoney,
+		Variables.varDefault('tournament_liquipediatier'),
+		placement.placeStart,
+		Variables.varDefault('tournament_type')
+	)
+	if opponent.opponentData.type == Opponent.solo then
+		lpdbData.participantflag = opponent.opponentData.players[1].flag
```

Comment:
Isn't this one done on commons already?

Reply:
right, this was added after i started building the custom module, removing now.